Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/SubgraphInterface.cpp

```diff
@@ -930,8 +936,10 @@ namespace ScriptCanvas
 
         AZStd::string ToString(const SubgraphInterface& subgraphInterface)
         {
+            auto nameOutcome = subgraphInterface.GetName();
+
             AZStd::string result("\n");
-            result += subgraphInterface.GetName();
+            result += nameOutcome.IsSuccess() ? nameOutcome.TakeValue() : "<un-named>";
```

Comment:
should this be handled explicitly, like report as error?

Reply:
Not in a tostring function, no